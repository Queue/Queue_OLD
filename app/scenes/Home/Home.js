//
// Home Scene

import React, { Component } from 'react';
import { View, Text} from 'react-native'

// Components
import PrimaryButton from '../../components/PrimaryButton';

export default class Home extends Component {
  constructor(props) {
    super(props);

    this.state = {};
  }

  render() {
    return (
      <View>
        <Text>Home Scene</Text>
      </View>
    );
  }
}
