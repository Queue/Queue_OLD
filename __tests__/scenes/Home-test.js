//
// Home scene test

import React from 'react';
import Home from '../../app/scenes/Home';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(
    <Home />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
