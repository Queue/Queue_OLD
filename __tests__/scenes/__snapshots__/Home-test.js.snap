exports[`test renders correctly 1`] = `
<View>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail">
    Home Scene
  </Text>
</View>
`;
